tutorials/terraform/data-proc-data-exchange-with-mch.tf

```
@@ -0,0 +1,281 @@
+# Infrastructure for Yandex Cloud Managed Service for ClickHouse cluster, Data Proc cluster and virtual machine.
+#
+# RU: https://cloud.yandex.ru/docs/data-proc/tutorials/exchange-data-with-mch
+#
+# Set the configuration of the Managed Service for ClickHouse cluster, Data Proc cluster and Virtual Machine.
+
+# Specify the pre-installation parameters:
+locals {
+  folder_id          = ""                              # Your folder ID.
+  network_id         = ""                              # Network ID for Managed Service for ClickHouse cluster, Data Proc cluster and VM.
```

Не уверен, но, кажется, правильно будет так:

```suggestion
  network_id         = ""                              # Network ID for the Managed Service for ClickHouse cluster, the Data Proc cluster and VM.
```